src/schemas/paths.json

```diff
@@ -37,7 +37,7 @@
               }
             },
             {
-              "$ref": "#/definitions/_asl_payload_template"
+              "$ref": "#/definitions/_payload_template_object"
```

Comment:
changed to target the object instead of the abstraction which allows for an array as well.